src/content/chainlink-local/build/ccip/foundry/local-simulator.mdx

```diff
@@ -0,0 +1,253 @@
+
+1. Invoke the `prepareScenario()` function to set up the necessary variables:
+
+   ```solidity
+   (Client.EVMTokenAmount[] memory tokensToSendDetails, uint256 amountToSend) = prepareScenario();
+   ```
+
+1. Record the initial token balances of Alice and Bob:
+
+   ```solidity
+   uint256 balanceOfAliceBefore = ccipBnMToken.balanceOf(alice);
+   uint256 balanceOfBobBefore = ccipBnMToken.balanceOf(bob);
+   ```
+
+1. Begin impersonating Alice to perform the subsequent actions:
+
+   ```solidity
+   vm.startPrank(alice);
+   ```
+
+1. Request 5 LINK tokens from the CCIP local simulator faucet for Alice:
+
+   ```solidity
+   ccipLocalSimulator.requestLinkFromFaucet(alice, 5 ether);
+   ```
+
+1. Construct the [Client.EVM2AnyMessage](/ccip/api-reference/client#evm2anymessage) structure with the receiver, token amounts, and other necessary details. We set data: `abi.encode("")` because we are not sending any arbitrary data (only tokens). Additionally, `extraArgs: Client._argsToBytes(Client.EVMExtraArgsV1({gasLimit: 0}))` is used because we are sending tokens to an EOA and do not expect any execution of receiver logic:
+
+   ```solidity
+   Client.EVM2AnyMessage memory message = Client.EVM2AnyMessage({
+       receiver: abi.encode(bob),
+       data: abi.encode(""),
+       tokenAmounts: tokensToSendDetails,
+       extraArgs: Client._argsToBytes(Client.EVMExtraArgsV1({gasLimit: 0})),
+       feeToken: address(linkToken)
+   });
+   ```
+
+1. Calculate the required fees for the transfer and approve the router to spend LINK tokens for these fees:
+
+   ```solidity
+   uint256 fees = router.getFee(destinationChainSelector, message);
+   linkToken.approve(address(router), fees);
+   ```
+
+1. Send the CCIP transfer request to the router:
+
+   ```solidity
+   router.ccipSend(destinationChainSelector, message);
+   ```
+
+1. Stop impersonating Alice:
+
+   ```solidity
+   vm.stopPrank();
+   ```
+
+1. Record the final token balances of Alice and Bob:
+
+   ```solidity
+   uint256 balanceOfAliceAfter = ccipBnMToken.balanceOf(alice);
+   uint256 balanceOfBobAfter = ccipBnMToken.balanceOf(bob);
+   ```
+
+1. Verify that Alice's balance has decreased by the amount sent and Bob's balance has increased by the same amount:
+
+   ```solidity
+   assertEq(balanceOfAliceAfter, balanceOfAliceBefore - amountToSend);
+   assertEq(balanceOfBobAfter, balanceOfBobBefore + amountToSend);
+   ```
+
+### Test case 2: Transfer with native gas fees
+
+The `test_transferTokensFromEoaToEoaPayFeesInNative` function tests the transfer of tokens between two externally owned accounts (EOA) while paying fees in native gas. Here are the steps involved in this test case:
+
+1. Invoke the `prepareScenario()` function to set up the necessary variables. (This step is the same as in the previous test case.)
```

Comment:
- Copy over steps from previous test case
- Tab the test cases (LINK gas fees, native gas fees) to reduce page length

Reply:
Disregarding because it's an "examine the code" section highlighting the differences between native and LINK test cases, not a set of steps users actually need to follow.